fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedTermVectorsFormatTest.java

```diff
@@ -77,4 +78,10 @@ public void tearDown() throws Exception {
     public void testMultiClose() throws IOException {
         BaseIndexFileFormatTestCaseUtils.testMultiClose(this);
     }
+
+    @Seeds({@Seed(), @Seed("F007433804DC4123")})
```

Comment:
Confirmed this ran twice with the same random seed, and the provided seed, as expected:
```
LuceneOptimizedTermVectorsFormatTest > testRandomExceptions > com.apple.foundationdb.record.lucene.codec.LuceneOptimizedTermVectorsFormatTest.testRandomExceptions {seed=[C185081D42F0F43C:F007433804DC4123]} STARTED

LuceneOptimizedTermVectorsFormatTest > testRandomExceptions > com.apple.foundationdb.record.lucene.codec.LuceneOptimizedTermVectorsFormatTest.testRandomExceptions {seed=[C185081D42F0F43C:F007433804DC4123]} PASSED

LuceneOptimizedTermVectorsFormatTest > testRandomExceptions > com.apple.foundationdb.record.lucene.codec.LuceneOptimizedTermVectorsFormatTest.testRandomExceptions {seed=[C185081D42F0F43C:A9AA66D9DCFEAD9C]} STARTED

LuceneOptimizedTermVectorsFormatTest > testRandomExceptions > com.apple.foundationdb.record.lucene.codec.LuceneOptimizedTermVectorsFormatTest.testRandomExceptions {seed=[C185081D42F0F43C:A9AA66D9DCFEAD9C]} PASSED

```